NeosModLoader/AssemblyHider.cs

```diff
@@ -60,16 +60,16 @@ private static void FindTypePostfix(ref Type? __result)
 					// an assembly was in neither neosAssemblies nor modAssemblies
 					// this implies someone late-loaded an assembly after NML, and it was later used in-game
 					// this is super weird, and probably shouldn't ever happen... but if it does, I want to know about it.
-					Logger.WarnInternal($"The \"{__result}\" type does not appear to part of Neos or a mod. It is unclear whether it should be hidden or not.");
+					// since this is an edge case users may want to handle it in different ways, the HideLateTypes nml config option allows them to chose.
```

Comment:
```suggestion
					// since this is an edge case users may want to handle in different ways, the HideLateTypes nml config option allows them to choose.
```